src/libs/web3/web3.connectors.ts

```diff
@@ -52,9 +59,18 @@ export const networkConnection: Connection = {
 // INJECTED CONNECTOR
 // ********************************** //
 
-const [web3Injected, web3InjectedHooks] = initializeConnector<MetaMask>(
-  (actions) => new MetaMask({ actions, onError })
-);
+const [web3Injected, web3InjectedHooks] = getInjectedProvider('ethereum')
+  ? initializeConnector<MetaMask>(
```

Comment:
not a fan of ternary operators for a non-trivial logic but if it's too annoying to initialize otherwise then ok